cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,26 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
+
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
```

Comment:
Ensure that the `db:clear` task is defined in your Cypress task configuration. This task should clear the database to ensure a clean state before each test.